cs/src/Connections/TunnelRelayTunnelHost.cs

```diff
@@ -495,11 +495,12 @@ void Session_Closed(object? sender, SshSessionClosedEventArgs e)
             // Reconnecting client session may cause the new session close with 'None' reason and null exception.
             if (cancellation.IsCancellationRequested)
             {
-                Trace.TraceInformation("Client ssh session cancelled.");
+                Trace.WithName("ClientSSH").Verbose("Session cancelled verbose.");
```

Comment:
Please don't add "verbose" suffix to the trace message. It's already traced as verbose via `.Verbose()` method.

Reply:
Sorry Yes, it was a copy paste mistake. I will fix it

Reply:
fixed